packages/manager/modules/telecom-spare/src/phones/phones.html

```diff
@@ -0,0 +1,35 @@
+<div class="telecom-phones-section">
+    <h1 data-translate="phones_spare_list_title"></h1>
+
+    <tuc-toast-message></tuc-toast-message>
+
+    <div class="oui-datagrid-topbar__actions" data-ng-if="!$ctrl.isLoading">
```

Comment:
Why not use the `extra-top` ?

Reply:
what it is?

Reply:
https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/datagrid > Extra top content

Reply:
Because I just want to add a button to refresh list of spare of phones